spec.md

```diff
@@ -210,6 +210,7 @@ creation to destruction.
 Figure 6: The lifecycle of a dynamically provisioned volume, from
 creation to destruction, when the Node Plugin advertises the
 STAGE_UNSTAGE_VOLUME capability.
+NOTE: When using Non-Graceful Node Shutdown feature in Kubernetes, status can be moved from PUBLISHED to CREATED without going through VOL_READY and/or NODE_READY status.
```

Comment:
This behavior is not introduced by Non-Graceful Node Shutdown feature.  Kubernetes already supports this behavior without Non-Graceful Node Shutdown. See "Test 2" in the PR description section below.  By forcefully deleting the Pods on the shutdown node manually, Kubernetes will behave this way.

https://github.com/kubernetes/kubernetes/pull/108486